cmd/fleetctl/users_test.go

```diff
@@ -29,3 +31,33 @@ func TestUserDelete(t *testing.T) {
 	assert.Equal(t, "", runAppForTest(t, []string{"user", "delete", "--email", "user1@test.com"}))
 	assert.Equal(t, uint(42), deletedUser)
 }
+
+func writeTmpCsv(t *testing.T, contents string) string {
+	tmpFile, err := ioutil.TempFile(t.TempDir(), "*.csv")
+	require.NoError(t, err)
+	_, err = tmpFile.WriteString(contents)
+	require.NoError(t, err)
+	return tmpFile.Name()
+}
+
+func TestCreateBulkUsers(t *testing.T) {
+	_, ds := runServerWithMockedDS(t)
+	ds.InviteByEmailFunc = func(ctx context.Context, email string) (*fleet.Invite, error) {
+		return nil, nil
+	}
+
+	csvFile := writeTmpCsv(t,
+		`Name,Email,Password,SSO,API Only,Global Role,Teams
+	user11,user11@example.com,P@ssw0rd!2,false,false,maintainer,
+	user12,user12@example.com,P@ssw0rd!2,false,false,admin,
+	user13,user13@example.com,P@ssw0rd!2,false,false,admin,1:team1
+	user14,user14@example.com,P@ssw0rd!2,false,false,,team14
+	user15,user15@example.com,P@ssw0rd!2,false,false,,1:admin`)
+
+	expectedText := `{"kind":"user_roles","apiVersion":"v1","spec":{"roles":{"admin1@example.com":{"global_role":"admin","teams":null},"user11@example.com":{"global_role":"observer","teams":null},"user12@example.com":{"global_role":"observer","teams":null},"user13@example.com":{"global_role":"observer","teams":null},"user14@example.com":{"global_role":"observer","teams":null},"user15@example.com":{"global_role":"observer","teams":null},"user1@example.com":{"global_role":"observer","teams":null},"user2@example.com":{"global_role":"observer","teams":null}}}}
```

Comment:
All returned roles have global `observer` role and `teams: null`.

Reply:
@lucasmrod Thanks for the great review, some of these i have mentioned in my last comment. Would you please give it a look. I was not able to affect the global role as well as the teams. That's why i have it like that till i figure it out. 
Would be great if you give it a look.

Reply:
Yes, see https://github.com/fleetdm/fleet/pull/3372#issuecomment-1001743899 (addressing the comments I added would fix the bugs).

Reply:
Hi @anelshaer! Did you have a chance to take a look at our comments? Let us know if something is not clear enough.

Sorry it took us several days to respond (mostly due to holidays).